packages/app/src/cli/constants.ts

```diff
@@ -87,8 +87,10 @@ export const activeUIExtensions = {
 export type UIExtensionTypes = typeof uiExtensions.types[number]
 
 export const uiExtensionTemplates = [
-  {name: 'React', value: 'react'},
-  {name: 'vanilla JavaScript', value: 'vanilla-js'},
+  {name: 'TypeScript', value: 'typescript'},
+  {name: 'Vanilla JavaScript', value: 'vanilla-js'},
```

Comment:
I'm starting to question wether we need `Vanilla` at this point. We could just go with `Javascript` now that we have `Typescript`

Reply:
Probably yes, maybe @MeredithCastile should confirm the specific names for this 🤔

Reply:
umm, I think it's ok if we remove the Vanilla everywhere, it's easy to understand:
- TypeScript
- JavaScript
- TypeScript React
- JavaScript React

What do you think?

Reply:
Let's update this to be a framework question. Like so:

> What framework would you like to use?
> - JavaScript, no additional framework
> - TypeScript
> - TypeScript React
> - JavaScript React

Reply:
I like the idea of reframing this question to be a framework question, but should we repeat the "no additional framework" like for `TypeScript` as well since that's also just a language choice?

Reply:
You've got me deep into [internet research](https://dev.to/bytebodger/is-typescript-really-a-language-ep1#:~:text=So%20apparently%20their%20survey%20endorses,lists%20TypeScript%20as%20a%20language.) on whether typescript is a language or a framework, @matteodepalo! I always understood TypeScript to be a validation "framework" on top of JavaScript, but I guess it's a "language" to most people.

What about getting rid of the noun altogether?

> What would you like to work in?
> - JavaScript
> - TypeScript
> - TypeScript React
> - JavaScript React